credential/reclaim/v0.1.0/schema.json

```diff
@@ -0,0 +1,133 @@
+    "appearance": {
+        "style": {
+            "color": "#4bbf95",
+            "icon": "./icon.svg"
+        }
+    },
+    "allOf": [
+        {
+            "$ref": "https://core.schemas.verida.io/base/latest/schema.json"
+        },
+        {
+            "properties": {
+                "appId": {
+                    "title": "App ID",
+                    "description": "Unique id of the task allocated by the allocator node",
+                    "type": "string"
+                },
+                "httpProviderId": {
+                    "title": "Http Provider Id",
+                    "description": "ID of http provider",
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "proofs": {
+                    "title": "Session Id",
+                    "description": "Values of public fields defined in schema",
+                    "type": "array",
+                    "items": {
+                        "type": "object",
+                        "properties": {
+                            "identifier": {
+                                "title": "Identifier",
+                                "description": "Identifier of provider",
+                                "type": "string"
+                            },
+                            "claimData": {
+                                "title": "Claim Data",
+                                "description": "Verifiable credential data",
+                                "type": "object",
+                                "properties": {
+                                    "provider": {
+                                        "title": "Protocol",
+                                        "description": "Protocol of request",
+                                        "type": "string"
+                                    },
+                                    "parameters": {
+                                        "title": "Parameters of request",
+                                        "description": "Stringfied JSON object of parameters and API reqeusts",
+                                        "type": "string"
+                                    },
+                                    "identifier": {
+                                        "title": "Identifier of request",
+                                        "description": "Identifier of request",
+                                        "type": "string"
+                                    },
+                                    "epoch": {
+                                        "title": "Epoch of request",
+                                        "description": "Epoch of request",
+                                        "type": "string"
+                                    }
+                                }
+                            },
+                            "signature": {
```

Comment:
For arrays, you can also specify `minItems`

Reply:
Done